client/app/account/user/support-level/support-level.html

```diff
@@ -0,0 +1,66 @@
+<div class="module-useraccount-sections-support-level">
+    <div class="tab-content">
+        <div data-ovh-alert="useraccount.alerts.dashboardAdvanced"></div>
+
+        <h2 class="oui-heading_2" data-translate="user_account_support_level_section_title"></h2>
+        <h3 class="oui-heading_3" data-translate="user_account_support_level_section_title_what"></h3>
+
+        <p>
+            <span data-translate="user_account_support_level_section_title_description_1"></span><sup aria-label="(1)">*</sup>
+            <span data-translate="user_account_support_level_section_title_description_2"></span><sup aria-label="(2)">**</sup>
+            <span data-translate="user_account_support_level_section_title_description_3"></span>
+        </p>
+
+        <div class="row pt-4">
+            <oui-tile class="col mb-4 mx-2 py-0"
```

Comment:
`.col` comes with default padding. I'm not sure that adding extra margins (`mx-2` here) will harmonize the UI.

Are they really needed?

Reply:
Due to `col` being apply directly on the `oui-tile`